### ISSUE TITLE
More Eldritch items

### ISSUE THREAD

**Issue by Fortanono** [Azanor/thaumcraft-suggestions#9]
The Eldritch path is really cool, but there aren't very many materials to go with. Really, there's Voidmetal and that's it. The most ridiculous things are the Void Thaumaturge's Robes, and the Primal Staff Core. One is making robes out of metal, and the other is a combination of the best things to make something even better. So I have some ideas.

**Oldewood Trees:** Have you ever wondered what kind of tree sap Amber comes from? Well, I thought, why not put Amber Blocks in the Crucible with Victus and Permutatio to get Ancient Resin, which you then infuse with a bunch of Greatwood Logs and Silverwood Leaves to make an Oldewood Sapling. The leaves do not drop new Saplings, but they do drop some more Amber to make more Saplings. This is used to make the new Staff Core and probably some other stuff (possibly new Clockwork stuff in the Eldritch tier?)

**Treasured/Kingly Fabric:** Like the Oldewood Trees, this started with me asking a question: We all know what is on the inside of treasure bags, but what about outside? Maybe putting a bag in an Arcane Workbench with Ordo and Perditio could net you some Commonly/Uncommonly/Rarely Treasured Fabric, and all three tiers combined would make Kingly Fabric, used for a new tier of Robes.

One more random idea: How about having Void Seeds be made with Quicksilver Drops instead of Seeds? Would make them more expensive, at least by a little bit.

**Comment by darthvader45** [comment]
This could be quite an update.  The Eldritch tab is somewhat bland as-is, and could use a bit of expansion.

**Comment by Fortanono** [comment]
As of now I'm just thinking of more materials. There should definitely be more content overall, but this is what I thought of for changes to the materials used.

**Comment by James-T-Oudvoure** [comment]
I'm in support.

**Comment by carso150** [comment]
Maybe not only expand the eldrich tab, add another tab related to the eldrich tab, like I don't know, a mental tab, based in the fact that the outer lands are only a mental proyection, and make more eldrich epiphanies

**Comment by Fortanono** [comment]
Alternatively, instead of doing an exact copy of what Thaumcraft did, have various paths you can take (like in issue #12). If you want more endgame content that expands on the tab, Azanor should put it in the Eldritch tab itself. After all, it's _far_ from a complete story.

**Comment by carso150** [comment]
We only scratch the begining, we are powerfull, we have conquered the overworld, but there is still something that is wrong, the outer lands are not the end of our journey, they are only the begining of something bigger, so much bigger

**Comment by Fortanono** [comment]
Exactly, but the way to do that is to expand the Eldritch Tab and not to create another tab that works the exact same way.

**Comment by James-T-Oudvoure** [comment]
I'm kinda with Fortanono on this. The Eldritch tab is downright minuscule, at least in usable content, compared to other tabs. Adding another tab full of stuff that would fit better in the Eldritch tab would be an utter waste.

**Comment by carso150** [comment]
I say make bigger the eldrich tab and add even more things in a second tab, that way we have to forbidden and hidden tabs, I even will say that we need an crimson tab, but thats for a new post that eventually I will do when I think better what he needs to have

**Comment by James-T-Oudvoure** [comment]
If we're going to have a second Eldritch tab, the current one'll have to be much bigger than it is now.

**Comment by Argis13** [comment]
A function of the Oldewood trees could be to slowly convert all the diamonds in the chunk to a new type ore that can form awesome robes(with the kingly fabric)/wand caps. Maybe Empowered Jewels?  
Edit: Now that I think about all the possibilities, this could be the start of a whole new branch of research.

**Comment by darthvader45** [comment]
Hmmm, this could be amazing to see.

**Comment by Argis13** [comment]
Indeed, this is one of the best posts on here. I bet this could be an update in and of itself. I just wish Az would tell us what he thinks.

**Comment by Azanor** [comment]
Some good suggestions here.
